src/world/ObjectMgr.h

```diff
@@ -270,6 +270,67 @@ enum AREATABLE_CATEGORY
     AREAC_SANCTUARY          = 6,
 };
 
+struct simpleEventScripts
```

Comment:
This does not comply with AscEmu coding standard 7.2, http://www.ascemu.org/wiki/index.php?title=Coding_standards#class.2Fstruct.2Fprotocol.2Ffunction.2Fmethod.2Fnamespace

Additionally, as the struct stores a single script, the name should be SimpleEventScript